### ISSUE TITLE
Window goes black when moving to a monitor other than my Main Monitor

### ISSUE THREAD

**Issue by CraigIAP** [LorenzCK/OnTopReplica#164]
I have 3 monitors. When using the application on my main monitor, it works perfectly. But if I drag it to one of my other two monitors, it just shows a black picture.

**Comment by VackerSimon** [comment]
Do you have several GPUs? For example an iGPU running the main monitor and a dedicated one for the rest?

**Comment by CraigIAP** [comment]
No, but I did "solve" the problem or at least find the culprit. My main monitor which was working has HDR. My other two do not. Once I turned off HDR on the main monitor, it works properly on all monitors. However, with HDR on, OnTopReplica still was not working even if I had the OnTop window AND the window I was trying to copy on the same monitor (One of my two side monitors)
